cmd/module/cmd.go

```diff
@@ -3,29 +3,29 @@ package main
 import (
 	"context"
 
+	"go.uber.org/zap/zapcore"
 	"go.viam.com/rdk/components/camera"
 	"go.viam.com/rdk/logging"
 	"go.viam.com/rdk/module"
 
 	"github.com/erh/viamrtsp"
+	"go.viam.com/utils"
 )
 
 func main() {
-	err := realMain()
-	if err != nil {
-		panic(err)
-	}
+	utils.ContextualMain(mainWithArgs, module.NewLoggerFromArgs("client"))
 }
-func realMain() error {
-
-	ctx := context.Background()
-	logger := logging.NewDebugLogger("client")
 
+func mainWithArgs(ctx context.Context, args []string, logger logging.Logger) error {
 	myMod, err := module.NewModuleFromArgs(ctx, logger)
 	if err != nil {
 		return err
 	}
 
+	if logger.Level() != zapcore.DebugLevel {
+		viamrtsp.SetLibAVLogLevelFatal()
```

Comment:
This gets rid of the spammy `no frame!` logs which don't impact video quality unless the module is configured to run in debug mode.

Reply:
If we merge in https://github.com/erh/viamrtsp/pull/5 before this change, this can be removed